modules/rosa-create-objects.adoc

```diff
@@ -109,78 +107,99 @@ Create a new cluster.
 .Syntax
 [source,terminal]
 ----
-$ rosa create cluster --cluster=<cluster_name> | <cluster_id> [arguments]
+$ rosa create cluster --cluster-name=<cluster_name> [arguments]
```

Comment:
There is, but I should note it in a comment for future writers as well :) thanks!

Reply:
I've noted the following in the main locations this parameter is explained:

`//Note to writers: The create cluster command specifically uses --cluster-name because a cluster ID does not exist yet. All other commands use --cluster because either the name or the ID can be used.`